packages/orchestration/src/examples/sendAnywhereFlows.js

```diff
@@ -0,0 +1,59 @@
+import { M, mustMatch } from '@endo/patterns';
+
+/**
+ * @import {Orchestrator, OrchestrationAccount} from '../types.js';
+ */
+
+const { entries } = Object;
+
+// in guest file (the orchestration functions)
+// the second argument is all the endowments provided
+
+export const orchestrationFns = harden({
```

Comment:
I'm not crazy about moving this to a separate file. Did we feel `sendAnywhere.contract.js` got too big, or was there a different motivation?

Reply:
Motivation is that it's a different runtime environment than the contract. Keeping the flows in a separate file will help prevent trying to access host values directly. Also it will let us improve linting. E.g. the "no async" lint rules need only apply to these files, not the contract.

Reply:
> Keeping the flows in a separate file will help prevent trying to access host values directly

Agree, but putting them at the top of the file also achieves the same

> the "no async" lint rules

Think you meant to say no `E`, but I see your point. Noting this can be achieved intra-file with something like `Property[key.name=/OfferHandler$/]` or `Property[key.name=/Handler$/]` if we enforce the convention (sendIt -> sendItHandler)